app/src/routes/lookup/how_to/+server.js

```diff
@@ -0,0 +1,52 @@
+import { PUBLIC_ONTOLOGY_API_HOST } from '$env/static/public'
+import { split_stem_and_sense } from '$lib/parser/token'
+import { json } from '@sveltejs/kit'
+
+
+/** @type {import('./$types').RequestHandler} */
+export async function GET({ url: { searchParams } }) {
+	/** @type {LookupTerm} */
+	const word = searchParams.get('word') ?? ''
+
+	let matches = (await get_matches_from_how_to(word)).map(transform_response)
+
+	return response({ term: word, matches })
+
+	/** @param {LookupResponse<HowToResult>} result  */
+	function response(result) {
+		const THREE_HOUR_CACHE = {
```

Comment:
what do you think about every 12 hours for this one since the automation only runs that often?

Reply:
Good point. I'll see if there's a way to say 'valid until'. If it gets cached an hour before the automation starts, we don't want it to be invalid for 11 hours.

Reply:
it might be hyper-optimizing at this point so I'm good with whatever you decide.